src/ui/public/vis/agg_config.js

```diff
@@ -265,7 +268,7 @@ class AggConfig {
     }
 
     if (!this.type) return '';
-    let pre = percentageMode ? 'Percentage of ' : '';
+    let pre = percentageMode ? this._percentageOfLabel : '';
```

Comment:
That's actually not working like this, since it assumes, that "percentage of" can always be prefixed (see next line of code). We will need to make `this.type.makeLabel(this)` a parameter to this message (or in case `percentageMode` is `false`, just use `makeLabel` instead.